swift/llm/sft.py

```diff
@@ -199,7 +234,8 @@ def llm_sft(args: SftArguments) -> Dict[str, Union[str, Any]]:
         push_to_hub=args.push_to_hub,
         resume_from_checkpoint=args.resume_from_checkpoint,
         ddp_backend=args.ddp_backend,
-        gradient_checkpointing=args.gradient_checkpointing,
+        gradient_checkpointing=False
+        if use_torchacc() else args.gradient_checkpointing,
```

Comment:
Write the modifications for gradient_checkpointing in the argument.py directly.

Reply:
fixed